lib/apiForSheets.js

```diff
@@ -58,6 +59,8 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
     });
     const commits = await context.github.pulls.listCommits(prParams);
     var allAuthors = [];
+    const onBoardingTeamMember = [teamLeads.onboardingTeam];
+    const lead = onBoardingTeamMember[0];
```

Comment:
The onboarding team lead should be gotten from `teamLeads.onboardingTeam`.

Reply:
@jameesjohn sir i had addressed the requested changes it would be great if you can review the same.